src/LibPQ.jl

```diff
@@ -185,10 +185,25 @@ function Connection(str::AbstractString; throw_error::Bool=true, kwargs...)
         end
     end
 
-    return handle_new_connection(
-        Connection(libpq_c.PQconnectdbParams(keywords, values, false); kwargs...);
-        throw_error=throw_error,
-    )
+    # Make the connection
+    newconn = Connection(libpq_c.PQconnectdbParams(keywords, values, false); kwargs...)
```

Comment:
My convention is to use `jl_conn` for `LibPQ.Connection` instances to avoid confusion with `Ptr{PGconn}` instances, so please change `newconn` to that.